tools/runners/run-app.js

```diff
@@ -569,14 +566,15 @@ Object.assign(AppRunner.prototype, {
       }
 
       if (self.recordPackageUsage) {
-        stats.recordPackages({
+        // Maybe this doesn't need to be awaited for?
+        await stats.recordPackages({
           what: "sdk.run",
           projectContext: self.projectContext
         });
       }
 
-      var bundleResult = Profile.run((firstRun?"B":"Reb")+"uild App", () => {
-        return bundler.bundle({
+      var bundleResult = await Profile.run((firstRun?"B":"Reb")+"uild App", async () => {
+        return await bundler.bundle({
```

Comment:
If there is an uncaught error while bundling, it sometimes logs the error and keeps bundling, instead of crashing like it is supposed to. You can reproduce this by adding `  throw new Error('linker failed');` to the top of the `fullLink` function in `linker.js`.